.ort.yml

```diff
@@ -17,10 +29,18 @@ curations:
       https://github.com/nexB/scancode-toolkit/blame/develop/README.rst#L168"
     reason: "DATA_OF"
     concluded_license: "CC0-1.0"
-  - path: "analyzer/src/funTest/assets/projects/synthetic/php-composer/{empty-des, lockfile, no-deps, with-provide, with-replace}/composer.phar"
-    comment: "This file 'composer.phar' is part of PHP Composer and includes a mapping from human readable strings to SPDX license ids."
+  - path: "reporter-web-app/src/config.js"
+    comment: "Configuration file for WebApp reporter defining colors to use for each SPDX license."
     reason: "DATA_OF"
-    concluded_license: "MIT"
+    concluded_license: "Apache-2.0"
+  - path: "README.md"
+    comment: "GPL match in the sentence 'See [no_gpl_declared.kts] ... that declare the GPL'."
```

Comment:
Either complete the sentence with "... that declare the GPOL are used" or add "...".

Reply:
Done.